packages/python-sqlalchemy/superfunctions_sqlalchemy/adapter.py

```
@@ -0,0 +1,571 @@
+            conn = connection
+            if conn is not None:
+                result = conn.execute(insert(table).values(**params.data))
+                inserted = None
+                inserted_primary_key = list(getattr(result, "inserted_primary_key", ()) or [])
+                if inserted_primary_key:
+                    inserted = self._fetch_one_by_primary_key(
+                        conn,
+                        table,
+                        {
+                            column.name: value
+                            for column, value in zip(table.primary_key.columns, inserted_primary_key)
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)